composer.json

```diff
@@ -4,7 +4,7 @@
 	"description": "FOF (Framework on Framework) is a Rapid Application Development framework for Joomla!",
 	"require" : {
 		"php": ">=5.3.4",
```

Comment:
Try this and redo the lock file as nothing with nikic/php-parser changed in the last update
```json
   "require": {
        "php": "^5.3.10|~7.0",
        "nikic/php-parser": "~2.1|~3.0"
    },
```